actions/dependabot-bot/README.md

```diff
@@ -0,0 +1,42 @@
+# dependadotnet -- Generator for dependabot.yml for .NET
```

Comment:
You should probably add a section here to describe using it as an action.

Reply:
Done